docs/Contributing.asciidoc

```diff
@@ -643,6 +643,8 @@ Run tests by spawning a container manually, e.g.:
 
 Replace `OPENQA_LOCAL_CODE` with the location where you have the openQA code.
 
+NOTE: `run-tests-within-container` runs with CONTAINER_TEST enabled by default. It is used to run scripts which are not meant to run inside the *openqa_devel* and the image itself does not contain any container engine. Consider disabling it to run any tests in a container.
```

Comment:
```suggestion
NOTE: `run-tests-within-container` runs with CONTAINER_TEST enabled by default. It is used to run scripts which are not meant to run inside *openqa_devel* and the image itself does not contain any container engine. Consider disabling it to run any tests in a container.
```

Reply:
And please wrap the lines consistently at 80 characters